pkg/eventbus/event_bus.go

```diff
@@ -35,19 +47,42 @@ func (eb *eventbus) Start(c *context.Context) {
 	// no need to call TopicInit now, we have fixed topic
 	eb.context = c
 
-	mqttURL := config.CONFIG.GetConfigurationByKey("mqtt.server")
 	nodeID := config.CONFIG.GetConfigurationByKey("edgehub.controller.node-id")
-	if mqttURL == nil || nodeID == nil {
+	if nodeID == nil {
 		panic("mqtt url or node id not configured")
```

Comment:
panic message should only keep the node id , no mqtt url.

Reply:
tks, i will fix it.